packages/plugin-conversation/src/share-activity.js

```diff
@@ -78,6 +78,20 @@ const ShareActivity = SparkPlugin.extend({
    * @returns {EventEmittingPromise}
    */
   add(file) {
+    let actions;
+
+    // ArrayBuffer might be located in an object with the actions.
+    if (has(file, `file`) && has(file, `actions`)) {
```

Comment:
differentiating blobs and arraybuffers like this feels really brittle. why is this necessary?